Clients/GUI/wwwroot/css/site.css

```diff
@@ -246,3 +196,33 @@ app {
     .cur-icon .gbp-icon {
         background-image: url(/images/currencies/gbp.png)
     }
+
```

Comment:
May use the following https://bootstrap-4.ru/docs/4.3.1/components/navbar/ instead of your css classes in site.css classes?